### PR TITLE
Add terminology to the README

### LINKED ISSUE #58: Terminology
I think to better communicate and more easily build an API and UI, we need to come up with a uniform set of terminology. Below is my (very much changable) suggestion based on what I've heard the community say the most:

- A ~~"chapter"~~ "collective" is one singular instance/deployment of Chapter
  *For example, Women who Code at the domain chapter.womenwhocode.com*
- A "user" is a person who authed to the chapter
  *For example, Kognise*
- A ~~"group"~~ "chapter" is a container for events, with a description and subscribers, and one or more "admins" who can manage it
  *For example, Women who Code NYC*
- A "event" is a single meetup that users can RSVP to, has a specific location and time, and has "organizers" 
  *For example, Women who Code NYC April Meetup*

(All the examples here are fictional and meaningless and probably super inaccurate)

What do you guys think of these - could they made more clear? Am I missing anything?

### PR DESCRIPTION
**Pull request by kognise**
This adds recommended terminology to the documentation and could close #58.

### CODE REVIEW

**Comment by allella** [comment]
#46 has group(s) changed to "chapter(s)", so that makes the core entity of the schema align with this vocabulary.

File: README.md [hunk 1]
```diff
@@ -36,6 +36,20 @@ We will focus on building an open API first. Then developers can use the API to
 - [Ideas for Version 1 or Version 2 of the App](https://github.com/freeCodeCamp/chapter/issues/1)
 - [Introduce Yourself / Volunteer Your Skills](https://github.com/freeCodeCamp/chapter/issues/11)
 
+### Terminology
+
+To better communicate and more easily build an API and UI, we've decided on a collection of terminology to describe Chapter in a clear way.
+
+- A "collective" is one singular instance/deployment of Chapter
+  *For example, Women who Code at the domain chapter.womenwhocode.com*
```
Comment (ScottBrenner) [inline README.md]:
```suggestion
  *For example, Women who Code at the domain `chapter.womenwhocode.com`*
```

[URLs are backtick'd above](https://github.com/freeCodeCamp/chapter#welcome-to-chapter), best to do so here as well.

**Comment by allella** [comment]
@kognise you able to include minor tweaks to this pull request without much effort?  Scott posted a minor tweak above and I want to add a couple Quick Links to the Readme.  I can do another PR, but figured I'd either have to sync up with you as an upstream or wait until this is merged.  Or, have you add it if that's not a Github PR no-no.

**Comment by allella** [comment]
@kognise Quincy used "organization" in his latest vision statement and suggested we go with that, so the Readme now says "organization" instead of "collective".  There might be a better word down the road, but for now it's clear enough for the purposes of talking about things in a standard way.

There's no schema entity at this highest level, so I don't see any changes there either. That work for you?

**Comment by nik-john** [comment]
@all-contributors please add @kognise  for code

**Pull request merged**